### PR TITLE
Add ONIE support on a new Lenovo NE10032 platform 

### PR DESCRIPTION
**Pull request by psarker**
Adding ONIE support on a new X86 based Lenovo platform(NE10032)

### CODE REVIEW

File: machine/lenovo/lenovo_ne10032/rootconf/sysroot-lib-onie/sysinfo-platform [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+get_serial_num()
```
Comment (cbrune) [inline machine/lenovo/lenovo_ne10032/rootconf/sysroot-lib-onie/sysinfo-platform]: you don't need this function, as this is the default behavior:

https://github.com/opencomputeproject/onie/blob/master/rootconf/grub-arch/sysroot-lib-onie/sysinfo-arch#L11-L21

Comment (psarker) [inline machine/lenovo/lenovo_ne10032/rootconf/sysroot-lib-onie/sysinfo-platform]: Thank you Curt. yes, this is not required.. I have removed it and placed another push ..

**Comment by psarker** [comment]
machine/lenovo/lenovo_ne10032/rootconf/sysroot-lib-onie/sysinfo-platform is removed .. thank you for pointing this. Cheers!.

**Comment by cbrune** [comment]
Thanks.

I squashed 80960ed into 5960c61 and pushed the result to master:

- 9940029 Add ONIE support to a new Lenovo X86 platform NE10032

**Pull request closed without merging**